lib/teiserver/battle/libs/balance_lib.ex

```diff
@@ -27,7 +27,7 @@ defmodule Teiserver.Battle.BalanceLib do
   # which one will get to pick first
   @shuffle_first_pick true
 
-  @default_balance_algorithm "loser_picks"
+  @default_balance_algorithm "default"
```

Comment:
this doesn't make sense anymore to keep this as a hardcoded value. Especially since there is already `get_default_algorithm/1` a bit below. You can modify this function to take some inputs and return the correct module.
This way you keep a clean separation between "choosing the algorithm" and "running it". Right now it's a bit muddy.